F1WM/Services/DriversService.cs

```diff
@@ -0,0 +1,27 @@
+using F1WM.ApiModel;
+using F1WM.Repositories;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace F1WM.Services
+{
+	public class DriversService : IDriversService
+	{
+		private readonly IDriversRepository repository;
+	
+		public async Task<Drivers> GetDrivers(string letter)
+		{
+			if (letter.Length != 1) return null;
```

Comment:
This is truly confusing. If an input is invalid, the API shouldn't effectively return 404, but something else, 400 preferably. It should be handled by a controller.